velox/docs/functions/presto/datetime.rst

```diff
@@ -106,6 +106,10 @@ Date and Time Functions
 
     Returns ``timestamp`` as a UNIX timestamp.
 
+.. function:: to_iso8601(x) -> varchar
```

Comment:
Functions must appear in alphabetical order. Please, double check and move as necessary.

Reply:
Done.